style.qmd

```diff
@@ -92,8 +92,8 @@ There's a few different options for `Ruff` for GitHub Actions. See [Ruff docs](h
 Some free resources for writing better code:
 
 - [Tidy design principles](https://design.tidyverse.org/) by Hadley Wickham
-- [Software Engineering at Google](https://abseil.io/resources/swe-book) curated by Titus Winters, Tom Manshreck and Hyrum Wright
-
+- [The Programmatic Programmer](https://pragprog.com/titles/tpp20/the-pragmatic-programmer-20th-anniversary-edition/) by David Thomas, Andrew Hunt
```

Comment:
You suggested this in a previous PR and I don't want to include it cause it's not an easily available online book